### ISSUE TITLE
Adding several loss (criteria)

### ISSUE THREAD

**Issue by amirshamaei** [ashleve/lightning-hydra-template#609]
Hi, I am working on adopting H+PL template for MRI reconstruction. I need to have several loss functions. What is the best routine for adding them to a PL module?

Currently, I added this piece of code to init(). However, I believe it can be more efficient and general. 

``` python 
        criterion_1 : Callable = torch.nn.MSELoss,
        criterion_2:  Callable = None,
        criterion_3: Callable = None,
```

**Comment by amirshamaei** [comment]
I made it a bit better as follows:

```python
criterions: List = [torch.nn.MSELoss],
```

and config file: 

```
criterions:
  - _target_: torch.nn.MSELoss
  - _target_: torch.nn.MSELoss
```

and the training step:

```python
for criterion in self.criterions:
    loss += criterion(output_imgs, target_img)
```
I appreciate if you suggest a better approach

**Comment by amirshamaei** [comment]
I made it a bit better as follows:

```python
criterions: List = [torch.nn.MSELoss],
```

and config file: 

```
criterions:
  - _target_: torch.nn.MSELoss
  - _target_: torch.nn.MSELoss
```

and the training step:

```python
for criterion in self.criterions:
    loss += criterion(output_imgs, target_img)
```
I appreciate if you suggest a better approach
